crates/runtime/src/forge_runtime_extension.rs

```diff
@@ -38,32 +39,28 @@ use cairo_lang_starknet::contract::starknet_keccak;
 use cairo_vm::vm::errors::hint_errors::HintError::CustomHint;
 use cheatnet::cheatcodes::spy_events::SpyTarget;
 use cheatnet::execution::cheated_syscalls::SingleSegmentResponse;
-use cheatnet::execution::contract_execution_syscall_handler::{
-    print, ContractExecutionSyscallHandler,
-};
+use cheatnet::execution::contract_execution_syscall_handler::print;
 use starknet::signers::SigningKey;
 
 mod file_operations;
 
 pub type ForgeRuntime<'a> =
-    ExtendedRuntime<RuntimeExtension<TestExecutionState<'a>, ContractExecutionSyscallHandler<'a>>>;
+    ExtendedRuntime<RuntimeExtension<TestExecutionState<'a>, IORuntime<'a>>>;
 
 pub struct TestExecutionState<'a> {
     pub environment_variables: &'a HashMap<String, String>,
     pub contracts: &'a HashMap<String, StarknetContractArtifacts>,
 }
 
 // This runtime extension provides an implementation logic for functions from snforge_std library.
-impl<'a> ExtensionLogic
-    for RuntimeExtension<TestExecutionState<'a>, ContractExecutionSyscallHandler<'a>>
-{
-    type Runtime = ContractExecutionSyscallHandler<'a>;
+impl<'a> ExtensionLogic for RuntimeExtension<TestExecutionState<'a>, IORuntime<'a>> {
+    type Runtime = IORuntime<'a>;
 
-    fn get_extended_runtime_mut(&mut self) -> &mut ContractExecutionSyscallHandler<'a> {
+    fn get_extended_runtime_mut(&mut self) -> &mut IORuntime<'a> {
```

Comment:
I think this is doable with a macro as well

Reply:
Not sure about that, maybe but it will be changing in the next PR so we can look into this later, it is not very heavy on boilerplate

Reply:
It is removed at all in the next PR, resolving